content/en/ecosystem/adopters.md

```diff
@@ -0,0 +1,44 @@
+[AppDirect](https://www.appdirect.com/)           | Collector                       |
+[Care.com](https://www.care.com)                  | Collector, Go, Java, JS, .NET   |
+[Cloud Scale](https://www.cloudscaleinc.com)      | Collector, Python    |
+[CNCF Jaeger project](https://jaegertracing.io)   | Collector, Go SDK               |  [blog post](https://medium.com/jaegertracing/jaeger-embraces-opentelemetry-collector-90a545cbc24)
+[Dynatrace](https://www.dynatrace.com/)           | Collector, Java, JS, Python, Go, .NET |
+[EcoBee](https://www.ecobee.com/)                 | Collector, Java                 |  [blog post](https://www.honeycomb.io/blog/bees-working-together-how-ecobees-engineers-adopted-honeycomb-for-visibility-into-system-optimization-and-customer-experience)
+[F5](https://www.f5.com/)                         | Collector.                      |
+[Google Cloud Platform](https://cloud.google.com) | Collector, Go, Java, JS, Python |
+[Grafana Labs](https://grafana.com/)              | Collector                       |
+[Honeycomb](https://honeycomb.io)                 | Go                              |  [blog post](https://www.honeycomb.io/blog/interview-with-honeycomb-engineer-chris-toshok-dogfooding-opentelemetry/)
+[Instana](https://www.instana.com)                | Collector, JS                   |  [docs](https://www.ibm.com/docs/en/obi/current?topic=apis-opentelemetry)
+[Lightstep](https://lightstep.com)                | Go, JS                          |
+[Logicmonitor](https://www.logicmonitor.com/)     | Collector, Java, Go, JS, Python, .Net |
+[Logz.io](https://logz.io)                        | Collector, Java, C++, Go, JS, K8s   |  [blog](https://logz.io/learn/opentelemetry-guide/)
+[Microsoft](https://www.microsoft.com/)           | Collector, C++, Go, Java, JS, Python, Ruby, .NET    | [Azure blog](https://techcommunity.microsoft.com/t5/azure-monitor/opentelemetry-azure-monitor/ba-p/2737823), [Dapr docs](https://docs.dapr.io/operations/monitoring/tracing/otel-collector/), [GitHub blog](https://github.blog/2021-05-26-why-and-how-github-is-adopting-opentelemetry/)
+[OrderMyGear](https://www.ordermygear.com/)       |                                 |
+[PITS Globale Datenrettungsdienste](https://www.pitsdatenrettung.de/) | Python      |
+[Red Hat](https://redhat.com/)                    | Collector, Operator             |  [website](https://docs.openshift.com/container-platform/4.12/distr_tracing/distr_tracing_arch/distr-tracing-architecture.html)
+[Sentry Software](https://sentrysoftware.com) | Collector, Java, Semantic Conventions | [blog post](https://www.sentrysoftware.com/blog/2022-07-19/why-sentry-software-chose-opentelemetry-for-hardware-sentry.html)
+[Shopify](https://www.shopify.com/)               | Collector, Go, Ruby             |
+[Splunk](https://www.splunk.com/)                 | Collector, Java, JS             |
```

Comment:
given that vendors are already listed in Vendors page should we remove them from here?

Reply:
+1 Agree. Vendors and Adopters are two different categories and lists.

Reply:
We should change the sentence at the top to call out this distinction as well. Or maybe also just change the name to end-user, so as to match how the CNCF classifies organizations?

Reply:
yes, let's remove vendors.